### DISCUSSION TITLE
what is inside ORB vocabulary file ?

### DISCUSSION THREAD

**Discussion by an99990** [stella-cv/stella_vslam#318] [Q&A] [answered]

**Comment by ymd-stella** [comment, marked as answer]
Insufficient explanation.

**Reply by an99990** [reply]
in the README, it is mentionned that we can download ORB vocabulary file. The question is what is in that file ? Does it need to be regenerated when we go to a new environment. ?

**Reply by ymd-stella** [reply]
No.
